README.md

```diff
@@ -22,30 +22,26 @@ The public endpoint for the Valorem Trade API is `https://trade.valorem.xyz`.
 There are two principal user roles in the Valorem Trade API:
 
 - **Maker**: Makers sign offers in response to requests for quotes (RFQs).
-  They are responsible for having the requisite assets when a taker optionally 
+  They are responsible for having the requisite assets when a taker optionally
   fills their signed offer. Makers are presently required to request access to
-  the maker API via the [Valorem discord](https://discord.gg/valorem).
+  the maker API via the [Valorem discord](https://discord.gg/valorem); set to be lifted pending the battle testing of our offer validation protocol.
 
 - **Taker**: Takers request quotes from makers and optionally
-  execute signed offers via the Seaport smart contracts. Takers are presently 
-  required to possess a [Valorem Access Pass](https://opensea.io/collection/valorem-access-pass) to access the API.
-
-These protections are in place to ensure that the API is not abused during the
-early access period.
+  execute signed offers via the Seaport smart contracts. Takers can get early access to releases of new features with a [Valorem Access Pass](https://opensea.io/collection/valorem-access-pass).
 
 ## TLS Certificate Authority
 
-The Valorem Trade API uses the GoDaddy Root TLS certificate authority (CA) to 
+The Valorem Trade API uses the GoDaddy Root TLS certificate authority (CA) to
 issue certificates; some protobuf clients may need to add this CA, which can be
-found [here](certs/trade.valorem.xyz.pem).
+found [here](https://github.com/valorem-labs-inc/trade-interfaces/blob/main/certs/trade.valorem.xyz.pem).
 
 ## ALPN
 
 The Valorem Trade API supports HTTP/2 via the `h2` ALPN protocol.
 
 ## Keepalives and timeouts
 
-The Valorem Trade API sends HTTP/2 keepalives every 75 seconds and times out 
+The Valorem Trade API sends HTTP/2 keepalives every 75 seconds and times out
```

Comment:
Would have made this PR easier to grok if the whitespace changes came in apart from content changes.